### DISCUSSION TITLE
Specification for xml file location?

### DISCUSSION THREAD

**Discussion by buhtz** [ximion/appstream#593] [Q&A] [answered]
I couldn't find a document specifying the file system location of the xml files.
The online generator tells me to use `/usr/share/metainfo`. Also Debian use that folder by policy.

Is there an official spec I can link to?

I am not sure yet but it seems that Arch do not follow the recommendation. It seems to use `/usr/share/app-info/xmls/`.

**Comment by ximion** [comment, marked as answer]
It's literally the very first section: https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#spec-component-location

And Arch Linux follows that as well, `/usr/share/app-info/xmls/` is a legacy path for catalog metadata, **not** MetaInfo files though. Those should not be mixed. To learn more about MetaInfo files and their location, check out the first link.
If Catalog data is the thing you have (I doubt it, since you were talking about an online generator, and catalog data is usually composed by the distribution), then you can have a look at https://www.freedesktop.org/software/appstream/docs/chap-CatalogData.html instead.
